Polish Fixes 5

Thommond opened the pull request:
- Edited assign_grades function so submissions would not be none if there was a student in the session but no submission.

- The edited minor HTML p tag error

- Fixed issues such as courseMan.html loop, UI button, unneeded span, etc

Thommond committed:
I made sure that if you view an assignment it does not equal none.

Thommond committed:
I fixed the looping issue on courseMan.html.

I removed the span on roster.html

I made the feedback have a varchar of 100 to limit text.

I started a message for user friendlyness.

Thommond committed:
I updated the HTML of the h3's so the links would appear.

Thommond committed:
I updated the tests to reflect the UI changes of the
buttons for course, session, and assignment manage.

BrendanHorst commented on portal/schema.sql:
```diff
@@ -74,6 +74,6 @@ CREATE TABLE submissions (
   student_id bigint REFERENCES users (id),
   grade integer,
-  feedback text,
+  feedback varchar(10),
```
why was this changed to varchar(10)?  I assume that most teachers would want their feedback to be a sentence at the very least.

BrendanHorst reviewed (changes requested):
Everything looks good, I just think that 10 characters is way too small for feedback.

Thommond committed:
Updated the schedule.html to have a schedule portion in the
table and a seperate title portion in the table. This was to
be more user friendly.

Updated the feedback in the submissions table in the schema.sql
file since 10 characters is way to little it is now 500.

Added a local passowords doc so that we can manage the portal
locally.
